### ISSUE TITLE
No link to "Modern APIs" information in StyleSheetList notebox

### ISSUE THREAD

**Issue by atkinsmedley** [mdn/content#30170]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/StyleSheetList

### What specific section or headline is this issue about?

This is about the Note box under the summary paragraphs.

### What information was incorrect, unhelpful, or incomplete?

There's a note about using "Modern APIs" instead of this, but no mention as to what those APIs might be. In poking around MDN, I haven't found any other explanation. I'm left confused.

### What did you expect to see?

I expected to see a link to something else in MDN that describes the more modern APIs. The link to stackoverflow.com is helpful for providing historical context for historians, but not for modern developers trying to develop stuff.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/stylesheetlist`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/StyleSheetList
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/stylesheetlist/index.md
* Last commit: https://github.com/mdn/content/commit/acfe8c9f1f4145f77653a2bc64a9744b001358dc
* Document last modified: 2023-07-07T07:19:19.000Z

</details>

**Comment by wbamberg** [comment]
Thanks for filing!

What do you expect to be able to do with a link to "Modern APIs"? The way I read this, it does not intend to offer alternatives to `StyleSheetList`, it's only providing some historical context for why it is the way it is.

Personally I don't think this is a very helpful note (because it's only historical, not actionable), and it would be more helpful instead to expand on "converted to an [Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array)" in the previous para (e.g. by using `Array.from()`) and/or linking to https://developer.mozilla.org/en-US/docs/Web/API/StyleSheetList#get_all_css_rules_for_the_document_using_array_methods, which shows another way to do it.

Still, these notes were added in a fairly recent PR (https://github.com/mdn/content/pull/22833) so maybe I'd be in a minority here. @Elchi3 WDYT?

**Comment by atkinsmedley** [comment]
The note makes it sound like there's a new API for accessing these in a much easier way. Is that not the case? If it's not, then I think it needs rewording. If it is, a link to information about that better way makes sense.

**Comment by Elchi3** [comment]
> Still, these notes were added in a fairly recent PR (#22833) so maybe I'd be in a minority here. @Elchi3 WDYT?

No, I think you are right @wbamberg. I think "Modern API" is misleading there. I would reword it like this:

> Note: Typically list interfaces like `StyleSheetList` wrap around {{jsxref("Array")}} types, so you can use Array methods on them. This is not the case here [due to historical reasons](https://stackoverflow.com/questions/74630989/why-use-domstringlist-rather-than-an-array/74641156#74641156). However, you can convert `StyleSheetList` to an array, see the example below.

**Comment by atkinsmedley** [comment]
This seems like a good solution.
